modules/custom/az_migration/src/Plugin/migrate/process/ParagraphsCalloutFieldMerge.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Drupal\az_migration\Plugin\migrate\process;
+
+use Drupal\migrate\MigrateExecutableInterface;
+use Drupal\migrate\ProcessPluginBase;
+use Drupal\migrate\Row;
+
+/**
+ * Configure Behavior for paragraphs.
+ *
+ * @MigrateProcessPlugin(
+ *   id = "paragraphs_callout_field_merge"
+ * )
+ */
+class ParagraphsCalloutFieldMerge extends ProcessPluginBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function transform($value, MigrateExecutableInterface $migrate_executable, Row $row, $destination_property) {
+    // Merging the data into one field.
+    $title_field = $row->getSourceProperty('title_field');
+    $field_uaqs_summary = $row->getSourceProperty('field_uaqs_summary');
+    $field_uaqs_callout_background = $row->getSourceProperty('field_uaqs_callout_background');
```

Comment:
I think it would be good to make these field names be configuration (using `$this->configuration`) to the process plugin so that the migration file lists the machine names rather than hardcoding them.

Ideally the `az_paragraph_callout.yml` file should list all the machine names the migration uses, for example

```
process:
  field_az_text_area/value: 
    plugin: paragraphs_callout_field_merge
    title: title_field
    text: field_uaqs_summary
    background: field_uaqs_callout_background
```
(or similar)

Reply:
Update as per the comment. But again the HTML structure and field values are designed as per the fields right.